app/src/main/kotlin/com/intive/picover/auth/di/AuthModule.kt

```diff
@@ -0,0 +1,17 @@
+package com.intive.picover.auth.di
+
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.ktx.Firebase
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+
+@Module
+@InstallIn(SingletonComponent::class)
+object AuthModule {
```

Comment:
will be here some additional providers later? if not maybe you could rename it to `AppModule` as you suggested in another pr?